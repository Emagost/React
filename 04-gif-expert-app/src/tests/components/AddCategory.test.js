import React from 'react';
import { shallow } from 'enzyme';
import { AddCategory } from '../../components/AddCategory';

describe('Test en <AddCategory/>', () => {
    
    const setCategories = () => {};

    test('should show corretly', () => {
        
        const wrapper = shallow(<AddCategory setCategories = {setCategories} />);
        expect(wrapper).toMatchSnapshot();

    })
    

})
