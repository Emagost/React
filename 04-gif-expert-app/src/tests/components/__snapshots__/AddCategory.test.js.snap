// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <AddCategory/> should show corretly 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
